src/engine_redraw.h

```diff
@@ -55,6 +56,8 @@ TbBool keeper_screen_redraw(void);
 void smooth_screen_area(unsigned char *a1, long a2, long a3, long a4, long a5, long a6);
 
 int get_place_terrain_pointer_graphics(SlabKind skind);
+
+void draw_creature_view_icons(struct Thing* creatng);
```

Comment:
if it's only meant to be used inside of the same .c the prototype shouldn't be in the .h
eighter move the function above where you're using it or place the prototype at the top
like where these 2 are
void redraw_isometric_view(void);
void redraw_frontview(void);

Reply:
Why shouldn't it be in the header file?

Reply:
well mostly to avoid cluttered header files so you can easily see what the entry points of the file are